### DISCUSSION TITLE
Is there any way to use quarkus redis annotations while integrating with Redisson ?

### DISCUSSION THREAD

**Discussion by nitinty** [quarkusio/quarkus#40116] [Q&A] [answered]
Is there any way to use quarkus redis annotations (@CacheResult, @CacheInvalidate, @CacheInvalidateAll, @CacheKey) while integrating with Redisson ? Please assist here.

[a comment by an automated account is not shown]

**Comment by Ladicek** [comment, marked as answer]
That is not possible out of the box. The only way would be to write your own caching provider.
